client_src/test/test_integration.py

```diff
@@ -2,7 +2,7 @@
 import os
 from uuid import uuid4
 
-from relation_engine_client import REClient
```

Comment:
This was confusing since `relation_engine_client` is a directory with an `__init__.py` that defined `__all__` with `main.REClient`